app/models/item.rb

```diff
@@ -51,4 +52,13 @@ def location_name
     "Unknown"
   end
 
+  def repair_condition
+    severity_index = 0
+    repairs.each do |a|
```

Comment:
Rather than iterating over ALL the repairs, perhaps we should create a scope that would only show the ones where the condition is not `Ready`. A similar example is here
https://github.com/dusty-wil/ReadyResponder/blob/development/app/models/repair.rb#L24